charts/fluent-operator/values.yaml

```diff
@@ -97,6 +97,8 @@ fluentbit:
   imagePullSecrets: []
   # - name: "image-pull-secret"
   secrets: []
+  # fluent-bit daemonset use host network
+  hostNetwork: false 
```

Comment:
@wanjunlei what's the previous default value for `hostNetwork`?
We should keep the default as before to avoid break anything

Reply:
The default value is `false`.